community/bolt/src/main/java/org/neo4j/bolt/v1/runtime/TransactionStateMachine.java

```diff
@@ -235,8 +234,7 @@ else if ( ROLLBACK.matcher( statement ).matches() )
                                 BoltResultHandle resultHandle = executeQuery( ctx, spi, statement, params, noop() );
                                 ctx.currentResultHandle = resultHandle;
                                 ctx.currentResult = resultHandle.start();
-                                ctx.currentTransaction = null; // Periodic commit will change the current transaction, so
-                                // we can't trust this to point to the actual current transaction;
+                                ctx.currentTransaction = spi.beginTransaction( ctx.securityContext );
```

Comment:
Is the last periodic commit transaction always closed here? Does it matter if it is not (i.e. is it fine for this to open a `PlaceboTransaction`)?